src/interfaces/api-filing-interface.ts

```diff
@@ -91,6 +91,9 @@ export interface ApiFilingIF {
       region: string // may be null
     }
 
+    // continuation in filings only
+    continuationIn?: any // some object
```

Comment:
Please add a comment to the "Continuation In in ledger" ticket to populate this object.

Reply:
Added a point in To Do section in the ticket [20808](https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/20808)